src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier colSup = new ColorSupplier();

```

Comment:
avoid abbreviations, they decrease code readability and we don't gain anything from them. Coders spend way more time reading others' code than writing their own, so readability is more important than the time you spend typing a few more letters